src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -125,13 +125,13 @@
 # microtrait_trait-displaynamelong column as the column description, and
 # microtrait_trait-value as the cell value
 _MICROTRAIT_TO_SYS_TRAIT_MAP = {
-    loader_common_names.MICROTRAIT_TRAIT_NAME: loader_common_names.SYS_TRAIT_ID,
+    # loader_common_names.MICROTRAIT_TRAIT_NAME: loader_common_names.SYS_TRAIT_ID,
```

Comment:
commented them out so that tests can pass. Those will be removed in the following PRs.